ee/desktop/user/notify/notify_linux.go

```diff
@@ -101,9 +100,7 @@ func (d *dbusNotifier) Execute() error {
 			actionUri := signal.Body[1].(string)
 
 			for _, browserLauncher := range browserLaunchers {
-				ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
-				defer cancel()
-				cmd, err := browserLauncher(ctx, actionUri)
+				cmd, err := browserLauncher(context.TODO(), actionUri)
```

Comment:
Can't move a timeout with a `defer cancel` to a new function because then calling `cmd.Start` at the end of that function makes it fail immediately. I don't think the timeout is super meaningful here so I've removed it rather than adding a lint exception.